src/matgl/ext/ase.py

```diff
@@ -119,8 +119,8 @@ def get_graph(self, atoms: Atoms) -> tuple[dgl.DGLGraph, torch.Tensor, list]:
         return g, lat, state_attr
 
 
-class M3GNetCalculator(Calculator):
```

Comment:
Why cause unnecessary backward incompatible changes? You can add a line `M3GNetCalculator = PESCalculator` to make sure that existing codes using these work. Right now, you are going to cause all kinds of havoc in pymatgen, matcalc, etc.

Reply:
@shyuep, I just added the M3GNetClass, which inherits PESCalculator for backward compatibility.